doc/dev/TESTING.md

```diff
@@ -12,6 +12,8 @@ There are a few CMake options to configure the F3D testing framework:
 
 ## Running the tests
 
+Before running tests you need to fetch [git LFS](https://git-lfs.com/) data, which contains testing data and baselines. It can be done by running `git lfs pull` from the source directory.
```

Comment:
it is done automatically if git lfs is installed when you clone.

Reply:
So maybe mention needing to have it installed here instead?

Reply:
looks fine indeed.